src/components/featured-maps-list/featured-maps-list-component.jsx

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cx from 'classnames';
 import { useSpring, animated } from 'react-spring';
-
+import styles from './featured-maps-list-styles.module';
 
 const FeaturedMapsListComponent = ({ 
   className,
   featuredMapsList,
   selectedSidebar,
   isLandscapeMode,
   isFullscreenActive,
-  handleFeaturedMapClick
+  handleFeaturedMapClick,
+  handleFeatureMapHover
 }) => {
   const isOpen = selectedSidebar === 'featuredMapsList';
   const slide = useSpring({
     from: { marginLeft: -400 },
     marginLeft: isOpen && !isLandscapeMode && !isFullscreenActive ? 0 : -400,
     delay: isOpen && !isLandscapeMode && !isFullscreenActive ? 400 : 0
   })
+
+  const defaultCardOpen = featuredMapsList[0];
+  const [cardSelected, setCardSelected] = useState(false);
+  const [cardOpen, setCardOpen] = useState(defaultCardOpen);
+
+  const mouseOver = (card) => { setCardOpen(card); handleFeatureMapHover(card.slug) };
+  const mouseOut = () => {
```

Comment:
I think I would get rid of the `mouseOut` functionality. As the list items `height` is changing whenever are hovered it could drive the interface to undesired behaviours. It also allows the user to move the mouse with out losing the selection (if not the user is forced to have the cursor over the list item to read the text.

__With `mouseOut`__

![Kapture 2019-08-20 at 11 11 38](https://user-images.githubusercontent.com/6906348/63334331-81391780-c33b-11e9-9d03-fd444e1aab85.gif)
)

__Without `mouseOut`__

![Kapture 2019-08-20 at 11 16 17](https://user-images.githubusercontent.com/6906348/63334586-0290aa00-c33c-11e9-936c-fa61d2eb6dda.gif)

By the way I think this list item changing `height` is not the best UX solution. Maybe a fixed `height` with the text entering from above would be smoother or maybe just adding some animation/transition to the growing `height`. But that's a question for @fannycc :)

Reply:
@fannycc what would be your take on that? ☝️

Reply:
Yes, I definitely like better your approach @weberjavi, thanks! I removed the `mouseOut` and it works smoother

Reply:
hey! so...
1. **about the without mouse0ut approach**, just wondering if it's intuitive that I need to click on it to select that card when the status is already open...also how can I collapse it again? I thought there wouldn't be a big problem that the user needed to leave the mouse on top of the card to get the preview, after all there isn't any other available interaction in the interface than the cards. I'm just a bit a concern of adding this feature which is not consistent with the rest of our hovers. There is a bug in the first option "with mouse0ut" it seems that it gets crazy...I would be happy to see it working with the preview just in hover and then play around and show it to others. 

2. And about the height of the card. If it feels jumpy or weird we can try to fix the same height for the cards...bearing in mind that we have different lengths for the content and some might have a lot of space. Happy to try it out tho! :) then we should fix the longest example and duplicate this behaviour to the map with the selected card. All the cards should have the same height no matter the interface where they appear.